src/azul/__init__.py

```diff
@@ -763,30 +763,30 @@ def _index_prefix(self) -> str:
     class Catalog:
         """
         >>> plugins = dict(metadata=dict(name='hca'), repository=dict(name='tdr_hca'))
-        >>> kwargs = dict(atlas='', plugins=plugins, sources='')
```

Comment:
The types passed to the `Catalog` constructor in the test do not match the type annotations in the class body. The plugins are JSON instead of instances of `Config.Catalog.Plugin`. We're essentially passing the wrong value and asserting the wrong result. Compare with how the class is instantiated in the `from_json` methods, which I'm presuming is the correct behavior.

Reply:
Resolved at PL. No action needed.